pkgs/development/python-modules/pyrad/default.nix

```diff
@@ -1,60 +1,58 @@
-{ buildPythonPackage
-, fetchFromGitHub
-, fetchpatch
-, lib
-, poetry-core
-, netaddr
-, six
-, unittestCheckHook
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  lib,
+  poetry-core,
+  netaddr,
+  six,
+  unittestCheckHook,
+  fetchPypi,
 }:
+let
+  netaddr_0_8_0 = netaddr.overridePythonAttrs (oldAttrs: rec {
```

Comment:
We normally never do this because if any other package carries netaddr, we immeadatly have a conflict. If the cli or something from the package is important, it should be exposed outside of pythonPackages.